src/adapters/pulseaudio.cpp

```diff
@@ -124,8 +126,8 @@ int pulseaudio::process_events() {
         if (!spec_s_name.empty()) {
           o = pa_context_get_sink_info_by_name(m_context, spec_s_name.c_str(), sink_info_callback, this);
           wait_loop(o, m_mainloop);
+          break;
         }
-        break;
       // get default sink
```

Comment:
Can you add `//FALLTHRU`, like in commit https://github.com/jaagr/i3ipcpp/commit/57e59bd8bad21fbb9d18bbb233ff33012c74efa9, some compilers don't like implicit fallthroughs.